src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -878,14 +880,22 @@ export default function MeasureList(props: {
             />
           </form>
         </div>
-        {featureFlags.MeasureListCheckboxes && featureFlags.associateMeasures && (
-          <div tw="justify-self-end p-3">
-            <AssociateCmsIdAction
+        <div tw="justify-self-end p-3">
+          {featureFlags.MeasureListCheckboxes &&
+            featureFlags.associateMeasures &&
+            !featureFlags.MeasureListButtons && (
+              <AssociateCmsIdAction
+                measures={selectedMeasures}
+                onClick={associateCmsId}
+              />
+            )}
+          {featureFlags.MeasureListButtons && (
```

Comment:
I think need to make sure check boxes are there too, right?

Reply:
...yeah